paasta_itests/paasta_serviceinit.feature

```diff
@@ -54,18 +56,29 @@ Feature: paasta_serviceinit
       And we wait for the chronos job stored as "myjob" to appear in the job list
       And we paasta_serviceinit emergency-start the service_instance "testservice.testinstance"
      Then the job stored as "myjob" is enabled in chronos
-      And the job has running tasks
+      And the job stored as "myjob" has running tasks
 
-  Scenario: paasta_serviceinit can run emergency-start on a disabled chronos job
+  Scenario: paasta_serviceinit cannot run emergency-start on a disabled chronos job
```

Comment:
this isn't really correct - we can run emergency-start on a disabled chronos job, it's just noop